include/aspect/assembly.h

```diff
@@ -240,8 +265,18 @@ namespace aspect
            * that correspond only to the variables listed in local_dof_indices
            */
           FullMatrix<double>          local_matrix;
+          std::vector<FullMatrix<double> >         local_matrices_int_ext;
```

Comment:
can you add a comment what these matrices are? What is the length of the vector?